winui/Avatar-view/customization.md

```diff
@@ -9,15 +9,15 @@ documentation: ug
 
 # Customization in WinUI AvatarView 
 
-The `AvatarView` control offers a range of customization options, including modifying its border, background, and more.
+The [SfAvatarView](https://www.syncfusion.com/winui-controls/avatar-view) control offers a range of customization options, including modifying its border, background, and more.
```

Comment:
@MadhumithanJayapathi Use this as class API link.

Reply:
@Manivannan-E Used class API links